web-client/src/ducks/auth/phone/functions.ts

```diff
@@ -0,0 +1,23 @@
+import { IOTPAuth, IPhoneNumberAuth } from 'src/ducks/auth/phone/types';
```

Comment:
we could use ./types right? but it achieves the same thing

Reply:
auto imports 😄

Reply:
oh okay 😅